plugin/evm/atomic/txpool/mempool.go

```diff
@@ -22,6 +22,12 @@ var (
 	ErrConflict        = errors.New("conflict present")
 	ErrInsufficientFee = errors.New("insufficient fee")
 	ErrMempoolFull     = errors.New("mempool full")
+
+	errsNotToDiscard = []error{
+		nil,
```

Comment:
nit: I have to admit `nil` here looks a bit weird

Reply:
Yeah I agree.. I slightly preferred the slice over:
`err == nil || errors.Is(err, ErrAlreadyKnown) || errors.Is(err, ErrMempoolFull)`
but I'm happy to go back to just specifying each if we feel like that is cleaner.

Reply:
kept the slice but removed `nil` from it :+1: